addons/l10n_rw/data/template/account.fiscal.position-rw.csv

```diff
@@ -0,0 +1,4 @@
+"id","name","sequence","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id"
+"fiscal_position_rw_national","National B2B","2","1","1","base.rw","","",""
+"fiscal_position_rw_international","International","3","","","","","VAT_S_IN_RW_18","VAT_S_EXPORT"
```

Comment:
Just noticed while testing this in combination with the enterprise branch, that this fiscal position is not automatically applied. 
In my experience the "international" is usually automatically applied.. is it part of the spec that it isn't automatically applied in this case?

Reply:
forgot to put the auto apply, nice catch !